assets/xml/objects/object_gi_bottle_16.xml

```diff
@@ -1,8 +1,8 @@
 ﻿<Root>
     <File Name="object_gi_bottle_16" Segment="6">
-        <DList Name="object_gi_bottle_16_DL_0009A0" Offset="0x9A0" />
-        <DList Name="object_gi_bottle_16_DL_000D18" Offset="0xD18" />
-        <Texture Name="object_gi_bottle_16_Tex_000D20" OutName="tex_000D20" Format="i8" Width="32" Height="32" Offset="0xD20" />
-        <Texture Name="object_gi_bottle_16_Tex_001120" OutName="tex_001120" Format="i4" Width="32" Height="32" Offset="0x1120" />
+        <DList Name="gGiSeahorseBottleGlassAndCorkDL" Offset="0x9A0" />
+        <DList Name="gGiSeahorseBottleEmptyDL" Offset="0xD18" />
+        <Texture Name="gGiSeahorseBottleCorkTex" OutName="gi_sea_horse_bottle_cork" Format="i8" Width="32" Height="32" Offset="0xD20" />
```

Comment:
You use `Seahorse` (one word) for the name, but `sea_horse` (two words) for the OutName. They should probably be consistent (and I think one word is correct).